source/includes/_sim-scale-guide.html.md

```diff
@@ -0,0 +1,350 @@
+```
+First, login into your Azure account, then deploy your simulation on Azure and
+start training using jobs & tasks.
+> Create a Job and run it on your pool
+
+```
+az batch job create --id <your-job-name> --pool-id <your-pool-id>
+```
+
+###Create a job
+Jobs are containing multiple tasks and run on your pool.
+
+>Copy and Paste into task.json
+
+```
+[ 
+
+         { 
+
+         "id": "simulation-task-1", 
+
+         "displayName": "Simulation", 
+
+         "commandLine": "python3 /simulation/<your-sim-start> --url <bonsai_url> --username <bonsai_username> --accesskey <bonsai_accesskey>", 
+
+         "constraints": { 
+
+             "maxTaskRetryCount": 0, 
+
+             "maxWallClockTime": "P10D" 
+
+         }, 
+
+         "containerSettings": { 
+
+             "containerRunOptions": null, 
+
+             "imageName": "<your-acr>.azurecr.io/<your-simulation>:1.0", 
+
+             "registry": null 
+
+         }, 
+
+         "environmentSettings":[] 
+
+     } 
+
+ ]
+```
+  
+### Create tour first task
+```
+az batch task create --job-id <your-job-name> --json-file ./task.json 
+```
+Use your preferred editor and create a file named “task.json”. This allows you
+to simplify managing parameters when creating tasks. Please copy and paste the
+ content to the right into your editor and update \<your-sim-start>,\<bonsai_url>,
+  \<bonsai_username>,\<bonsai_accesskey,\<your-sim-start>,\<your-acr>.
+
+Run the `az batch` command, it will create a task with the id:“simulation-task-1”.
+
+Create additional tasks by modifying the task.json file as needed.
+
+More background on jobs and tasks can be found here:
+<https://docs.microsoft.com/en-us/azure/batch/scripts/batch-cli-sample-run-job>
+
+You can create additional jobs using the “Create Job” step with a different
+“job-id” and then use the “az batch task create …” command to assign a task to
+that newly created job. This could be useful if you want to associate jobs with
+differently configured pools.
+
+## View progress of your training
+
+Visit <https://beta.bons.ai>, go to your dashboard and select the BRAIN you are
+currently training. It’s recommended to validate that data from the simulator is
+being fed into the Brain by monitoring the simulation tab of the training graph.
+At this point, training status should have changed to “Simulator: Connected”.
+
+## Check log files
+
+If the simulator won’t connect and no data is arriving at the BRAIN, please
+review the log files associated with your Azure Batch pool. Go to
+<https://portal.azure.com> and navigate to your Azure Batch Pool. Select the
+node in your pool and enter the “files” section. If anything went wrong, error
+files should show up at this location and provide more details of the problem.
+This assumes that your locally executed container worked, and you’ve created a
+pool based on a single node.  
+On <http://beta.bons.ai> you will see standard out from one of the simulators
```

Comment:
Is this true? My understanding was that stdout only shows up for managed sims...

Reply:
It is. I'll verify again but it showed up last time I ran it.